app/src/main/java/com/github/se/signify/ui/screens/home/QuestScreen.kt

```diff
@@ -195,14 +224,22 @@ fun QuestDescriptionDialog(
           Row(
               modifier = Modifier.fillMaxWidth(),
               horizontalArrangement = Arrangement.SpaceBetween) {
-                Button(
-                    onClick = { isFingerspellVisible = true },
-                    colors =
-                        ButtonDefaults.buttonColors(
-                            containerColor = MaterialTheme.colorScheme.secondary,
-                            contentColor = MaterialTheme.colorScheme.onSecondary)) {
-                      Text(stringResource(R.string.try_fingerspell_button_text))
-                    }
+                if (isCompleted) {
+                  Text(
+                      text = "Completed",
```

Comment:
You should move this hardcoded value to the file strings.xml

Reply:
Good catch!